web/client/utils/LayersUtils.js

```diff
@@ -294,13 +295,23 @@ const LayersUtils = {
         return mapState;
     },
     geoJSONToLayer: (geoJSON, id) => {
+        const bbox = toBbox(geoJSON);
         return {
             type: 'vector',
             visibility: true,
             group: 'Local shape',
             id,
             name: geoJSON.fileName,
             hideLoading: true,
+            bbox: {
+                bounds: {
+                    minx: bbox[0],
+                    miny: bbox[1],
+                    maxx: bbox[2],
+                    maxy: bbox[3]
+                },
+                crs: "EPSG:4326"
```

Comment:
Are you sure the bbox is always EPSG:4326? Have you tried with data in other coordinate systems?

Reply:
I have tested shapefile with EPSG: 3003 and EPSG: 32632.
Always return EPSG: 4326

Reply:
good